metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/ElasticSearchGraphService.java

```diff
@@ -113,6 +114,10 @@ public void addEdge(@Nonnull final Edge edge) {
         count
     );
 
+    if (Objects.isNull(response)) {
```

Comment:
nit: I think response == null is more readable. Objects.isNull should only be used as function inputs for stream functions.